README.md

```diff
@@ -1,6 +1,6 @@
-# Valhdiem Server with CDK
+# Valhiem Server with CDK
 
-An "easy" way to setup the infrastructure and a running valheim server (with manual backup capabilities) on AWS!
+An "easy" way to setup the infrastructure and a running Valheim server (with manual backup capabilities) on AWS!
```

Comment:
Let's leave these out of the PR.

I changed the header spelling so please rebase. :+1: